test/parallel/test-runner-mocking.js

```diff
@@ -1054,3 +1057,19 @@ test('setter() fails if getter options is true', (t) => {
     t.mock.setter({}, 'method', { getter: true });
   }, /The property 'options\.setter' cannot be used with 'options\.getter'/);
 });
+
+test('wrong import syntax should throw error after module mocking.', () => {
+  const { stdout, stderr } = child.spawnSync(
+    process.execPath,
+    [
+      '--experimental-test-module-mocks',
+      '--experimental-default-type=module',
+      fixtures.path('module-mocking/wrong-import-after-module-mocking.js'),
+    ]
+  );
```

Comment:
Can you use either `require('../common/child_process').spawnSyncAndExitWithoutError` or `require('../common').spawnPromisified` please?

Reply:
I will revise it as you suggested.

Reply:
Thank you for your specific and kind reviews on behalf of all the comments.